src/Forms/DropdownField.php

```diff
@@ -124,6 +126,39 @@ public function getHasEmptyDefault()
         return parent::getHasEmptyDefault() || $this->Required();
     }
 
+    /**
+     * Provides <option> list for the template
+     * This allows for a faster rendering of dropdowns with lots of options
+     *
+     * @return string
+     */
+    public function renderOptionsHTML(): string
```

Comment:
Not sure here but does it need to be public? People can override the `Field` method if needed.
```suggestion
    private function renderOptionsHTML(): string
```